### ISSUE TITLE
[Bug]: Very poor color contrast for default text

### ISSUE THREAD

**Issue by nk9** [CaiJimmy/hugo-theme-stack#754]
### What happened?

The unselected colors of the default sidebar text/background in the theme are #bababa/#f5f5fa in light mode. This has a contrast ratio of just 1.8:1, which is far below the necessary ratio of 4.5:1 for regular text. (See this [summary of WCAG requirements](https://webaim.org/standards/wcag/checklist#g1.4).)

Here's how the text looks today:

<img width="126" alt="image" src="https://user-images.githubusercontent.com/3646730/210824473-82d5f00d-fcb8-4d8f-bc06-367d0408a737.png">

And how it would look with a compliant contrast ratio (#707070/#f5f5fa):

<img width="118" alt="image" src="https://user-images.githubusercontent.com/3646730/210825062-9f3f3b7e-d2a0-48eb-b218-b4ba48bc6dea.png">

You can use the [Color Contrast Analyzer tool](https://www.tpgi.com/color-contrast-checker/) to explore color contrast for text. Here's one that [integrates into Chrome](https://accessibleweb.com/color-contrast-checker/).

On a related note, the docs don't discuss overriding colors. I added the below to my `custom.scss` to change my site's colors to be compliant in light mode, but it would be helpful if the correct solution for this pretty common desire were given in the docs.

```scss
:root {
    --body-text-color: #707070;
    --card-text-color-tertiary: #767676;

    li.current span {
    	font-style: italic;
    }
}
```

### Hugo version

0.109.0

### Theme version

3.16

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### Link to the page with bug

https://demo.stack.jimmycai.com/

**Comment by CaiJimmy** [comment]
Sorry for the late reply. I'm still tied up with academic tasks and can't find many times to work on this project.

**This looks good to me!** Would you like to create a pull request and submit it? (I'll review it once I have more time).

Just one little thing, is there any particular reason that you add a custom font style to the menu item?

**Comment by nk9** [comment]
My thinking on that was that the contrast between the current item and the newly-dark standard text was itself quite low. So I added the italic font style as a "current" signifier. It could be anything, but I think there should be some further differentiator.

Happy to make a PR!
